Formula/mongoose.rb

```diff
@@ -3,8 +3,7 @@ class Mongoose < Formula
   homepage "https://github.com/cesanta/mongoose"
   url "https://github.com/cesanta/mongoose/archive/6.17.tar.gz"
   sha256 "5bff3cc70bb2248cf87d06a3543f120f3b29b9368d25a7715443cb10612987cc"
-  # license ["GPL-2.0", "Cesanta"] - pending https://github.com/Homebrew/brew/pull/7953
-  license "GPL-2.0"
+  license any_of: ["GPL-2.0", "Cesanta"]
```

Comment:
```suggestion
  license "GPL-2.0-only"
```

From the [`LICENSE` file](https://github.com/cesanta/mongoose/blob/34d4fb8cf03cdd74773f594aaa1502811841f195/LICENSE#L15-L16):

> Alternatively, you can license this software under a commercial
> license, as set out in <https://www.cesanta.com/license>.

Given that it's an alternative and it's a commercial license, we only need to list the `GPL` one.